in_toto/cmd/root.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "in-toto",
+	Short: "Framework to secure integrity of software supply chains",
+	Long:  `A framework to secure the integrity of software supply chains https://in-toto.io/`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+	},
+}
+
+var layoutPath string
+var pubKeyPaths []string
+var linkDir string
+
+func init() {
+	rootCmd.PersistentFlags().StringVarP(&layoutPath,
+		"layout", "l", "",
+		`Path to root layout specifying the software supply chain to be verified`)
+	rootCmd.PersistentFlags().StringSliceVar(&pubKeyPaths,
+		"layout-keys", []string{},
+		`Path(s) to PEM formatted public key(s), used to verify the passed 
+root layout's signature(s). Passing at	least one key using
+'--layout-keys' is	required. For each passed key the layout 
+must carry a valid signature.`)
+	rootCmd.PersistentFlags().StringVarP(&linkDir,
+		"link-dir", "d", "",
```

Comment:
I think these flags should live on the verify subcommand.  Other in-toto commands don't need these flags.